pro/src/screens/IndividualOffer/StocksEventEdition/StocksEventEdition.tsx

```diff
@@ -193,11 +234,15 @@ const StocksEventEdition = ({
     currentSortingColumn,
```

Comment:
La vérité c'est que si on veut faire des bons patterns on n'a pas besoin de useEffect ici : https://react.dev/learn/you-might-not-need-an-effect#sharing-logic-between-event-handlers
Ça c'est la vraie solution, car tout le code qu'on execute dans ce useEffect est une réaction à un event utilisateur, donc on ne devrait pas utiliser d'effect

Reply:
Du coup si on veut corriger correctement, soi on fait cette solution soit on la fait pas mais la solution avec des memos/callback est pas le bon pattern :)